### DISCUSSION TITLE
ActivityPub / Fediverse support for blogs, websites

### DISCUSSION THREAD

**Discussion by jawadakhan97** [quarto-dev/quarto-cli#8826] [Feature Requests]
### Description

Hello, WordPress recently-ish introduced ActivityPub support, it would be nice for a blog or website to have posts pushed onto the fediverse, like on Mastodon or Lemmy.

I tend to get most blog posts linked from Mastodon, it would he wonderful if I could subscribe directly to a quarto blog or website rather than follow a person with all their content, or create elaborate rules to filter what I want.

Here are some relevant resources...
https://github.com/BasixKOR/awesome-activitypub
https://seb.jambor.dev/posts/understanding-activitypub/
https://dev.to/wadecodez/how-to-join-the-fediverse-wout-mastodon-1agh

**Comment by mcanouil** [comment]
You should be able to integrate this as a Quarto extension, see <https://quarto.org/docs/extensions/creating.html>.

For example, related extensions:
- https://github.com/schochastics/quarto-social-share
- https://github.com/sellorm/quarto-social-embeds
- https://github.com/shafayetShafee/collapse-social-embeds

**Comment by AndreasThinks** [comment]
I built a version of this that kind of works.

https://github.com/AndreasThinks/quarto-mastodon-comments
